src/main/java/is/ulrik/SciViewTutorial/HelloWorldPlugin.java

```diff
@@ -1,5 +1,12 @@
 package is.ulrik.SciViewTutorial;
 
+import org.scijava.Context;
```

Comment:
Filthy eclipse user, revert this or be gone!

Reply:
I'm so sorry... please except my apologies... 
Your devoted friend,
Florian